### PR TITLE
fixes rows referenced before assignment error in get_response_inv

### PR DESCRIPTION
**Pull request by derick-hess**

### CODE REVIEW

**Comment by derick-hess** [comment]
There may be more fixes to on this to come after Steve is done. Tested on a local copy of 15-016

File: kitchen/ph5tostationxml.py [hunk 1]
```diff
@@ -440,7 +440,9 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
     
     def get_response_inv(self, obs_channel):
         das = obs_channel.data_logger.serial_number
-        self.ph5.read_das_t(das, reread=False)
+        start = (obs_channel.start_date - datetime.datetime(1970,1,1)).total_seconds()
+        end = (obs_channel.end_date - datetime.datetime(1970,1,1)).total_seconds()
+        self.ph5.read_das_t(das, start, end,  reread=False)
```
Comment (nick-iris) [inline kitchen/ph5tostationxml.py]: Although this is correct, I would avoid using `datetime.datetime(1970,1,1)` and instead use `datetime.fromtimestamp(mktime(gmtime(0)))`.

ie.
```
def get_epoch_seconds(date_to_convert):
    return (date_to_convert - datetime.fromtimestamp(mktime(gmtime(0)))).total_seconds()
```
command line
```
>>> datetime.fromtimestamp(mktime(gmtime(0)))
datetime.datetime(1970, 1, 1, 0, 0)
```

Comment (nick-iris) [inline kitchen/ph5tostationxml.py]: Quick side note:

Eventually I think we should consolidate a lot of the commonly used logic, like the code above, from ph5tomsAPI.py, ph5tostationxml.py, and ph5toexml.py into a ph5utils.py module.

By creating a utilities module we avoid maintaining multiple instances of the same code. For example the ph5tostationxml.py and ph5toexml.py modules both use identical copies of the radial and rectangular geographic coordinate methods.

**Comment by nick-iris** [comment]
I think this is okay to merge. If we use this frequently throughout the codebase I'll create a new method in the ph5utils module.

**Pull request merged**